src/objectTranslations/Plan_Requirement__c-ca.objectTranslation

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+    <caseValues>
+        <plural>false</plural>
+        <value><!-- Plan Requirement --></value>
```

Comment:
Did you intend to add this, or will we be updating it before merging?

Reply:
This is what the Platform generated because we enabled translation bench.